tests/test_mathematical.py

```diff
@@ -2151,6 +2151,14 @@ def test_ediff1d_errors(self):
         to_end = dpnp.array([5], dtype="f4")
         assert_raises(TypeError, dpnp.ediff1d, a_dp, to_end=to_end)
 
+        # another `to_begin` sycl queue
+        to_begin = dpnp.array([-20, -15], sycl_queue=dpctl.SyclQueue())
```

Comment:
Shouldn't that tests be part of `test_sycl_queue.py` scope?

Reply:
I would leave it here because `test_sycl_queue.py` is used to test function works with only valid queues. 
We do not check there errors with different queues